homeassistant/util/color.py

```diff
@@ -436,10 +436,12 @@ def color_rgbw_to_rgb(r: int, g: int, b: int, w: int) -> tuple[int, int, int]:
 
 
 def color_rgb_to_rgbww(
```

Comment:
Should we rename this to have `kelvin` in it instead and make the old one wrap the new one so any components that call it with `mired` values keep working during transition?

Reply:
I did some digging and I couldn't find any custom components calling this.